src/aequitas_webapp/views.py

```diff
@@ -164,7 +164,7 @@ def audit_file(name, dirname):
                       attr_cols=group_variables)
 
     (_gv_df, report) = audit(df,
-                             model_id=1,
+                             # model_id=1,
                              configs=configs,
```

Comment:
are we passing model_id ?

Reply:
we should NOT be passing model_id, it breaks the webapp. Internal calculations and error handling take care of this now.